odoo/addons/base/models/ir_actions_report.py

```diff
@@ -97,7 +97,7 @@ class IrActionsReport(models.Model):
         ' browser PDF means the report will be rendered using Wkhtmltopdf and'
         ' downloaded by the user.')
     report_name = fields.Char(string='Template Name', required=True,
-                              help="For QWeb reports, name of the template used in the rendering. The method 'render_html' of the model 'report.template_name' will be called (if any) to give the html. For RML reports, this is the LocalService name.")
+                              help="For QWeb reports, name of the template used in the rendering. The method 'render_html' of the model 'report.template_name' will be called (if any) to give the html.")
```

Comment:
```suggestion
                              help="For QWeb reports, name of the template used in the rendering. The method 'render_html' of the model 'report.template_name' will be called (if any) to provide the HTML.")
```

Reply:
Actually, if we only allow QWeb reports now, no need to specify it in the massge

Reply:
In that case we can remove the help from here.